### DISCUSSION TITLE
Exchanging observed data files between projects

### DISCUSSION THREAD

**Discussion by wilbertdew** [Open-Systems-Pharmacology/Forum#940] [Questions & Problems] [answered]
Hi all,

Is there a way to exchange observed data between two different Mobi models without the need for reimporting the data from an excel file?
I tried saving the observed data as .pkml file, but I cannot import these files in either Mobi or PK-Sim.
Thanks for your help!

Best,

Wilbert

**Comment by PavelBal** [comment, marked as answer]
Hi Wilbert,

you definitely should be able to import exported pkml in MoBi:

![image](https://user-images.githubusercontent.com/25903040/125770389-79a6ef0f-80ba-4914-8719-126479da0103.png)

This feature is unfortunately not available in PK-Sim for now.

**Comment by wilbertdew** [comment]
Thanks Pavel!

**Comment by prvmalik** [comment]
Be careful that there is a bug where the molecular weight gets divided by 1000 or 1000000 when importing between projects. This is a property of the observed data file and is in the top left. It affects conversion from umol to ug, for example.

**Reply by Yuri05** [reply]
@prvmalik do we have a bug report for that? If no: could you create one?

**Reply by prvmalik** [reply]
Yes if it happens again I will record the steps that occurred to reproduce, will not make a bug report until then. I have been playing around in 9.1 and not able to reproduce for now.
